scripts/common.Makefile

```diff
@@ -0,0 +1,42 @@
+# Globals to be included into all Makefiles
+# specification of the used openapi-generator-cli (see also https://github.com/ITISFoundation/openapi-generator)
+
+REPO_ROOT               := $(realpath $(dir $(abspath $(lastword $(MAKEFILE_LIST))))/..)
```

Comment:
NOTE that ``$(CURDIR)`` is part of `Makefile` for current's file directory! https://www.gnu.org/software/make/manual/html_node/Quick-Reference.html

Reply:
Right, but I seem to encounter issues when this one is included into other make files. Basically when I use `$(CURDIR)` here I get a different `REPO_ROOT` in the root Makefile and the clients/python Makefile. That's why I did it like this.